### PR TITLE
Adds Musicbrainz tags to lastfm plugin, PSA test, and plugin reload command 

### PR DESCRIPTION
**Pull request by mattbox**
- Added code to use artist genre tags from Musicbrainz instead of the ones that Last.fm gives for response to the now playing command.
- Test file for PSA plugin
- Added `load` command to the botui module to reload plugins without restarting the bot
- Some minor code cleanup, formatting, and commenting

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@ test.ini
 what.ini
 cookies.jar
 *.json
+!user_*.json
+!track*.json
```
Comment (thomwiggers) [inline .gitignore]: I'd prefer `!test/fixtures/*.json` as that makes it more explicit what's going on.

File: onebot/plugins/acl.py [hunk 2]
```diff
@@ -77,7 +77,7 @@ class ACLPlugin(object):
 
     Configuration settings:
         - ``superadmin``: username to store in the database with
-        ``all_permissions`. username depends on mask chosen.
+        - ``all_permissions``: username depends on mask chosen
```
Comment (thomwiggers) [inline onebot/plugins/acl.py]: `all_permissions` isn't a setting, it's the permission given to the name in `superadmin`.

File: onebot/plugins/botui.py [hunk 3]
```diff
@@ -132,6 +132,15 @@ def quote(self, mask, target, args):
         self.bot.send(cmd)
 
     @command(permission='admin', show_in_help_list=False)
+    def load(self, mask, target, args):
```
Comment (thomwiggers) [inline onebot/plugins/botui.py]: Does this also work for plugins that are not already loaded? If not, there probably should be some error handling. If so, the help-text possibly needs to be updated.

Comment (thomwiggers) [inline onebot/plugins/botui.py]: If it does work for 'new' loads, maybe change the help-text to `(re)loads`.

File: onebot/plugins/lastfm.py [hunk 4]
```diff
@@ -306,13 +314,28 @@ def fetch_extra_trackinfo(self, username, info):
...
-                self.log.warning("Tags is not a list: %r", taglist)
-            else:
-                info['tags'] = [tag['name'] for tag in taglist]
+        # if 'toptags' in api_result and 'tag' in api_result['toptags']:
+        #     taglist = api_result['toptags']['tag']
+        #     if not isinstance(api_result['toptags']['tag'],
+        #                       list):  # pragma: no cover
+        #         self.log.warning("Tags is not a list: %r", taglist)
+        #     else:
+        #         info['tags'] = [tag['name'] for tag in taglist]
```
Comment (thomwiggers) [inline onebot/plugins/lastfm.py]: Why comment this out. Either delete it or (alternatively) turn it into a configurable thing: either use mbrainz tags or use last.fm tags.

File: tests/test_plugin_lastfm.py [hunk 5]
```diff
@@ -327,6 +327,8 @@ def wrap():
             assert not mock.called, "Shouldn't call get_info if not recent"
         self.bot.loop.run_until_complete(wrap())
 
+#####
```
Comment (thomwiggers) [inline tests/test_plugin_lastfm.py]: Why this addition?

**Review by thomwiggers — CHANGES_REQUESTED** [review]
Overall I'm pretty happy with this change (and fairly excited for musicbrainz support 😃 ) but there are some small things that I think need to be fixed.

File: onebot/plugins/botui.py (hunk 3 above)
Comment (mattbox) [inline onebot/plugins/botui.py]: I added some error handling to limit it to plugins that have already been loaded. I believe it's possible to load new plugins, but it's not really what the `reload()` method was intended for. I added this mainly for debugging and changing settings.

Comment (thomwiggers) [inline onebot/plugins/botui.py]: Please rename the command / function to `reload` then for clarity.

**Review by thomwiggers — CHANGES_REQUESTED** [review]
I think with this last small change it can be merged.

**Comment by thomwiggers** [comment]
Thanks a lot, I should bump releasing a new version higher up my todo list...

**Pull request merged**